.github/workflows/notebook_ci.yaml

```diff
@@ -64,6 +64,8 @@ jobs:
         pip install .
         cd ..
         python -m pip install git+https://github.com/qutip/qutip-qip
+        python -m pip install jax jax[cpu] diffrax equinox
```

Comment:
These requirements for jax seem to already be in the setup.cfg. Is it still necessary to add them here?